bin/dartdoc.dart

```diff
@@ -249,104 +255,108 @@ main(List<String> arguments) async {
-  parser.addFlag('version',
-      help: 'Display the version for $name.', negatable: false);
   parser.addFlag('add-crossdart',
       help: 'Add Crossdart links to the source code pieces.',
       negatable: false,
       defaultsTo: false);
-  parser.addFlag('sdk-docs',
-      help: 'Generate ONLY the docs for the Dart SDK.', negatable: false);
-  parser.addFlag('show-warnings',
-      help: 'Display warnings.', negatable: false, defaultsTo: false);
-  parser.addFlag('show-progress',
-      help: 'Display progress indications to console stdout', negatable: false);
-  parser.addOption('sdk-readme',
-      help: 'Path to the SDK description file.  Deprecated (ignored)');
-  parser.addOption('input', help: 'Path to source directory.');
-  parser.addOption('output',
-      help: 'Path to output directory.', defaultsTo: defaultOutDir);
-  parser.addMultiOption('header',
-      splitCommas: true, help: 'paths to header files containing HTML text.');
+  parser.addOption('ambiguous-reexport-scorer-min-confidence',
+      help:
+          'Minimum scorer confidence to suppress warning on ambiguous reexport.',
+      defaultsTo: "0.1",
+      hide: true);
+  parser.addFlag('auto-include-dependencies',
+      help:
+          'Include all the used libraries into the docs, even the ones not in the current package or "include-external"',
+      negatable: false,
+      defaultsTo: false);
+  parser.addMultiOption('category-order',
+      help:
+          'A list of package names to place first when grouping libraries in packages. '
+          'Unmentioned categories are sorted after these. (deprecated, replaced by package-order)',
+      splitCommas: true);
+  parser.addOption('example-path-prefix',
+      help: 'Prefix for @example paths.\n(defaults to the project root)');
+  parser.addMultiOption('exclude',
+      splitCommas: true, help: 'Library names to ignore.');
+  parser.addMultiOption('exclude-packages',
+      splitCommas: true, help: 'Package names to ignore.');
+  parser.addOption('favicon',
+      help: 'A path to a favicon for the generated docs.');
   parser.addMultiOption('footer',
       splitCommas: true, help: 'paths to footer files containing HTML text.');
   parser.addMultiOption('footer-text',
       splitCommas: true,
       help: 'paths to footer-text files '
           '(optional text next to the package name and version).');
-  parser.addMultiOption('exclude',
-      splitCommas: true, help: 'Library names to ignore.');
-  parser.addMultiOption('exclude-packages',
-      splitCommas: true, help: 'Package names to ignore.');
+  parser.addMultiOption('header',
+      splitCommas: true, help: 'paths to header files containing HTML text.');
+  parser.addFlag('help',
+      abbr: 'h', negatable: false, help: 'Show command help.');
+  parser.addFlag('hide-sdk-text',
+      help:
+          "Drop all text for SDK components.  Helpful for integration tests for dartdoc, probably not useful for anything else.",
```

Comment:
Single-quotes here too?

Reply:
There's a lint for that 😄

Reply:
@kevmoo Indeed, but there's apparently no way to automatically apply the lint resolution to all the lints that show up when I enable it, and doing Double-Click Alt-Enter Down-arrow Down-arrow Enter several hundred times does not sound like my idea of fun.

In the meantime I'll try not to make things worse.  Done.